projects/apache-commons-lang/StringEscapeUtilsFuzzer.java

```diff
@@ -0,0 +1,33 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import org.apache.commons.lang3.StringEscapeUtils;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import java.util.regex.*;
+
+public class StringEscapeUtilsFuzzer {
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+
+        String safeHtml = StringEscapeUtils.escapeHtml3(data.consumeString(50));
+        assert !safeHtml.contains("</script") 
```

Comment:
Is there some seed corpus/dictionary we can provide to increase the probability of triggering these?

Reply:
Sure. I'll add that and split the fuzz target for each type of corpus